src/components/SceneApp/Pages/Workloads.ts

```diff
@@ -96,13 +102,11 @@ export function getClusterByWorkloadScene() {
             promDSVar.changeValueTo(newPromDs.uid);
           }
         } catch (e) {
-          trackException({
-            exception: e instanceof Error ? e : new Error(stringify(e)),
-            severityLevel: SeverityLevel.Error,
-            properties: {
-              reporter: "Scene.Main.WorkloadsScene",
-              action: "changePromVariableOnClusterChange"
-            }
+          telemetryClient.reportException("grafana_plugin_promdsvarchange_failed", {
+            reporter: reporter,
+            exception: "e instanceof Error ? e : new Error(stringify(e))",
```

Comment:
here also

Reply:
whoops, good catch, stringified it for testing.. removed all of them